src/styles/utils/_extends.scss

```diff
@@ -0,0 +1,8 @@
+%absolute-bottom-left {
+  position: absolute;
+  bottom: 50%;
+  left: 50%;
+
+  transform: translateX(-50%);
+  transform-origin: bottom;
+}
```

Comment:
your clock hands are not exactly in the middle, you can fix it by adding flex justify center to relative block